config/settings.yml

```diff
@@ -4,6 +4,7 @@ teacher_training_api:
   # URL of the API app (teacher-training-api)
   base_url: http://localhost:3001
 current_cycle: 2021
+expand_university: false # When true HEI institutions have search radii increased. See https://bat-design-history.netlify.app/find-teacher-training/search-results-locations/
```

Comment:
you probably want a hard test for this
ie

```ruby
expect(subject.expand_university).to be_falsey
```

Reply:
I've updated the specs to assert `false` rather than the result of `Settings.expand_university`